components/proxy/src/test/java/com/hotels/styx/admin/handlers/MetricsHandlerTest.java

```diff
@@ -44,6 +44,6 @@ public void respondsToRequestWithJsonResponse() {
     public void exposesRegisteredMetrics() {
         metricRegistry.counter("foo").inc();
         FullHttpResponse response = waitForResponse(handler.handle(get("/metrics").build()));
-        assertThat(response.bodyAs(UTF_8), is("{\"version\":\"3.0.0\",\"gauges\":{},\"counters\":{\"foo\":{\"count\":1}},\"histograms\":{},\"meters\":{},\"timers\":{}}"));
+        assertThat(response.bodyAs(UTF_8), is("{\"version\":\"3.1.3\",\"gauges\":{},\"counters\":{\"foo\":{\"count\":1}},\"histograms\":{},\"meters\":{},\"timers\":{}}"));
```

Comment:
Just curious as why it is reporting version 3.1.3 when we are upgrading to 3.2.6?

Reply:
I assumed it's JSON scheme version: com.codahale.metrics.json.MetricsModule#VERSION

Reply:
It's definitely not the library version as that one was 3.1.1 before, but the JSON version was 3.0.0